RELEASE-NOTES.md

```diff
@@ -105,6 +110,15 @@ mkdir /tmp/node/db
 
 #### Start node
 
+It is recommended that at least 1024 files can be opened in the session where you start the node: this is needed not only for files on disk (e.g. persisted chain) but also for network connections with peers.
+You can check your max number of open files:
+* On Ubuntu, by running `ulimit -n`;
+* On macOS, by running `ulimit -n`.
+
+If your max number of open files is lower than 1024, you are recommended to raise such configuration:
+* On Ubuntu, `ulimit -n 1024` shall be sufficient;
+* On macOS, `ulimit -n 1024` shall be sufficient; if not sufficient on your OS, please report also the output of `sysctl -a | grep files`.
```

Comment:
This worked for me. Please test locally too though.

Reply:
Example:
```
$ sysctl -a | grep files
kern.maxfiles: 49152
kern.maxfilesperproc: 24576
kern.num_files: 4884
```